### PR TITLE
feat(textile,api,ui,data)!: update durability index computation

### PR DESCRIPTION
**Pull request by n1k0**
[user story](https://www.notion.so/Durabilit-Suppression-de-crit-res-mati-res-et-dur-e-de-commercialisation-371f0f41e03f446683a7f8c6ab952552)

### CODE REVIEW

**Review by ccomb — APPROVED** [review]
LGTM except the new checkbox in the story : `service de réparation ou de garantie`→`service de réparation`

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "test:backend": "pipenv run pytest",
     "test:client": "NODE_ENV=test npm run db:build && npx elm-test",
     "test:review": "npx elm-review",
-    "test:server": "npm run server:build && jest",
+    "test:server": "npm run server:build && NODE_ENV=test jest",
```
Comment (vjousse) [inline package.json:42]: Why not setting the env var for the whole command? `NODE_ENV=test npm run server:build && jest`

**Comment by n1k0** [comment]
@ccomb as I commented on the card and [on the mattermost thread](https://mattermost.incubateur.net/fabnum-mte/pl/nanmghj7zpddxfby1sfzmn151w), I think the checkbox thing should be another card as it doesn't have any connection with the tasks at hand.

**Review by vjousse — APPROVED** [review]
The US is not enough for me to understand the reasoning behind the new computations, so I can only approve the code but not the business logic. LGTM :+1:

File: package.json (hunk 1 above)
Comment (n1k0) [reply package.json:42]: because `jest` wouldn't see it, that's the way env var definition scopes work

**Comment by n1k0** [comment]
sorry I was confused reading the diff on mobile, scratch my comment

**Pull request merged**